diff -r -U 3 minecraft/net/minecraft/src/CodecMus.java minecraft_patched/net/minecraft/src/CodecMus.java
--- minecraft/net/minecraft/src/CodecMus.java
+++ minecraft_patched/net/minecraft/src/CodecMus.java
@@ -5,6 +5,7 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.net.URL;
 import java.net.URLConnection;
 import paulscode.sound.codecs.CodecJOrbis;
 
@@ -23,4 +24,7 @@
 			return null;
 		}
     }
+
+    private URL url;
+    private URLConnection urlConnection = null;
 }
