--- a/net/minecraft/client/Minecraft.java
+++ b/net/minecraft/client/Minecraft.java
@@ -1161,28 +1161,24 @@
       this.systemTime = 0L;
    }
 
-   private void preloadWorld(String loadingMessage) {
-      this.loadingScreen.printText(loadingMessage);
+   private void preloadWorld(String pString) {
+      this.loadingScreen.printText(pString);
       this.loadingScreen.displayLoadingString("Building terrain");
-      short short1 = 128;
+      char c = '\200';
       int i = 0;
-      int j = short1 * 2 / 16 + 1;
+      int j = (c * 2) / 16 + 1;
       j *= j;
-
-      for(int k = -short1; k <= short1; k += 16) {
+      for(int k = -c; k <= c; k += 16) {
          int l = this.theWorld.spawnX;
          int i1 = this.theWorld.spawnZ;
-         if (this.thePlayer != null) {
-            l = (int)this.thePlayer.posX;
-            i1 = (int)this.thePlayer.posZ;
+         if(thePlayer != null) {
+            l = (int)thePlayer.posX;
+            i1 = (int)thePlayer.posZ;
          }
-
-         for(int j1 = -short1; j1 <= short1; j1 += 16) {
-            this.loadingScreen.setLoadingProgress(i++ * 100 / j);
+         for(int j1 = -c; j1 <= c; j1 += 16) {
+            this.loadingScreen.setLoadingProgress((i++ * 100) / j);
             this.theWorld.getBlockId(l + k, 64, i1 + j1);
-
-            while(this.theWorld.updatingLighting()) {
-            }
+            while(theWorld.updatingLighting()) ;
          }
       }
 
