--- a/net/minecraft/client/Minecraft.java
+++ b/net/minecraft/client/Minecraft.java
@@ -1148,8 +1148,30 @@
       this.systemTime = 0L;
    }
 
-   private void preloadWorld(String param1) {
-      // $FF: Couldn't be decompiled
+   private void preloadWorld(String pString) {
+      this.loadingScreen.printText(pString);
+      this.loadingScreen.displayLoadingString("Building terrain");
+      char c = '\200';
+      int i = 0;
+      int j = (c * 2) / 16 + 1;
+      j *= j;
+      for(int k = -c; k <= c; k += 16) {
+         int l = this.theWorld.spawnX;
+         int i1 = this.theWorld.spawnZ;
+         if(thePlayer != null) {
+            l = (int)thePlayer.posX;
+            i1 = (int)thePlayer.posZ;
+         }
+         for(int j1 = -c; j1 <= c; j1 += 16) {
+            this.loadingScreen.setLoadingProgress((i++ * 100) / j);
+            this.theWorld.getBlockId(l + k, 64, i1 + j1);
+            while(theWorld.updatingLighting()) ;
+         }
+      }
+
+      this.loadingScreen.displayLoadingString("Simulating world for a bit");
+      j = 2000;
+      this.theWorld.dropOldChunks();
    }
 
    public void installResource(String soundName, File file) {
