--- a/net/minecraft/src/EntitySorter.java
+++ b/net/minecraft/src/EntitySorter.java
@@ -9,7 +9,7 @@
       this.entityForSorting = entity;
    }
 
-   public int compare(WorldRenderer rendererOne, WorldRenderer rendererTwo) {
+   public int sortByDistanceToEntity(WorldRenderer rendererOne, WorldRenderer rendererTwo) {
       return rendererOne.distanceToEntitySquared(this.entityForSorting) < rendererTwo.distanceToEntitySquared(this.entityForSorting) ? -1 : 1;
    }
 
