--- a/net/minecraft/src/EntitySorter.java
+++ b/net/minecraft/src/EntitySorter.java
@@ -9,7 +9,7 @@
       this.entityForSorting = entity;
    }
 
-   public int compare(WorldRenderer worldRenderer, WorldRenderer worldRenderer1) {
+   public int sortByDistanceToEntity(WorldRenderer worldRenderer, WorldRenderer worldRenderer1) {
       return worldRenderer.distanceToEntitySquared(this.entityForSorting) < worldRenderer1.distanceToEntitySquared(this.entityForSorting) ? -1 : 1;
    }
 
