--- a/net/minecraft/src/EntitySorter.java
+++ b/net/minecraft/src/EntitySorter.java
@@ -9,7 +9,7 @@
       this.field_1594_a = entity;
    }
 
-   public int compare(WorldRenderer worldRenderer, WorldRenderer worldRenderer1) {
+   public int func_1063_a(WorldRenderer worldRenderer, WorldRenderer worldRenderer1) {
       return worldRenderer.func_1202_a(this.field_1594_a) < worldRenderer1.func_1202_a(this.field_1594_a) ? -1 : 1;
    }
 
