--- a/net/minecraft/src/GLAllocation.java
+++ b/net/minecraft/src/GLAllocation.java
@@ -30,7 +30,7 @@
 
    public static synchronized void deleteTexturesAndDisplayLists() {
       for(int i = 0; i < displayLists.size(); i += 2) {
-         GL11.glDeleteLists(displayLists.get(i), displayLists.get(i + 1));
+         GL11.glDeleteLists(((Integer)displayLists.get(i)).intValue(), ((Integer)displayLists.get(i + 1)).intValue());
       }
 
       IntBuffer intbuffer = createDirectIntBuffer(textureNames.size());
@@ -38,7 +38,7 @@
       GL11.glDeleteTextures(intbuffer);
 
       for(int j = 0; j < textureNames.size(); ++j) {
-         intbuffer.put(textureNames.get(j));
+         intbuffer.put((Integer)textureNames.get(j));
       }
 
       intbuffer.flip();
