--- a/net/minecraft/src/NextTickListEntry.java
+++ b/net/minecraft/src/NextTickListEntry.java
@@ -37,7 +37,7 @@
       return this;
    }
 
-   public int compareTo(NextTickListEntry nextTickListEntry) {
+   public int comparer(NextTickListEntry nextTickListEntry) {
       if (this.scheduledTime < nextTickListEntry.scheduledTime) {
          return -1;
       } else if (this.scheduledTime > nextTickListEntry.scheduledTime) {
