--- a/net/minecraft/src/Packet60Explosion.java
+++ b/net/minecraft/src/Packet60Explosion.java
@@ -4,6 +4,7 @@
 import java.io.DataOutputStream;
 import java.io.IOException;
 import java.util.HashSet;
+import java.util.Iterator;
 import java.util.Set;
 
 public class Packet60Explosion extends Packet {
@@ -44,8 +45,10 @@
       int i = (int)this.explosionX;
       int j = (int)this.explosionY;
       int k = (int)this.explosionZ;
+      Iterator iterator = this.destroyedBlockPositions.iterator();
 
-      for(ChunkPosition chunkposition : this.destroyedBlockPositions) {
+      while(iterator.hasNext()) {
+         ChunkPosition chunkposition = (ChunkPosition)iterator.next();
          int l = chunkposition.x - i;
          int i1 = chunkposition.y - j;
          int j1 = chunkposition.z - k;
