--- a/net/minecraft/src/Packet60.java
+++ b/net/minecraft/src/Packet60.java
@@ -4,6 +4,7 @@
 import java.io.DataOutputStream;
 import java.io.IOException;
 import java.util.HashSet;
+import java.util.Iterator;
 import java.util.Set;
 
 public class Packet60 extends Packet {
@@ -33,8 +34,24 @@
 
    }
 
-   public void writePacketData(DataOutputStream var1) throws IOException {
-      // $FF: Couldn't be decompiled
+   public void writePacketData(DataOutputStream dataOutputStream) throws IOException {
+      dataOutputStream.writeDouble(field_12236_a);
+      dataOutputStream.writeDouble(field_12235_b);
+      dataOutputStream.writeDouble(field_12239_c);
+      dataOutputStream.writeFloat(field_12238_d);
+      dataOutputStream.writeInt(field_12237_e.size());
+      int i = (int)field_12236_a;
+      int j = (int)field_12235_b;
+      int k = (int)field_12239_c;
+      int j1;
+      for(Iterator iterator = field_12237_e.iterator(); iterator.hasNext(); dataOutputStream.writeByte(j1)) {
+         ChunkPosition chunkposition = (ChunkPosition)iterator.next();
+         int l = chunkposition.x - i;
+         int i1 = chunkposition.y - j;
+         j1 = chunkposition.z - k;
+         dataOutputStream.writeByte(l);
+         dataOutputStream.writeByte(i1);
+      }
    }
 
    public void processPacket(NetHandler netHandler) {
