--- a/net/minecraft/src/Packet60Explosion.java
+++ b/net/minecraft/src/Packet60Explosion.java
@@ -4,6 +4,7 @@
 import java.io.DataOutputStream;
 import java.io.IOException;
 import java.util.HashSet;
+import java.util.Iterator;
 import java.util.Set;
 
 public class Packet60Explosion extends Packet {
@@ -44,16 +45,16 @@
       int i = (int)this.explosionX;
       int j = (int)this.explosionY;
       int k = (int)this.explosionZ;
-
-      for(ChunkPosition chunkposition : this.destroyedBlockPositions) {
+      int j1;
+      for(Iterator iterator = this.destroyedBlockPositions.iterator(); iterator.hasNext(); dataOutputStream.writeByte(j1)) {
+         ChunkPosition chunkposition = (ChunkPosition)iterator.next();
          int l = chunkposition.x - i;
          int i1 = chunkposition.y - j;
-         int j1 = chunkposition.z - k;
+         j1 = chunkposition.z - k;
          dataOutputStream.writeByte(l);
          dataOutputStream.writeByte(i1);
          dataOutputStream.writeByte(j1);
       }
-
    }
 
    @Override
