--- a/net/minecraft/src/RecipeSorter.java
+++ b/net/minecraft/src/RecipeSorter.java
@@ -10,7 +10,7 @@
       this.craftingManager = craftingManager;
    }
 
-   public int compare(CraftingRecipe recipeOne, CraftingRecipe recipeTwo) {
+   public int compareRecipes(CraftingRecipe recipeOne, CraftingRecipe recipeTwo) {
       if (recipeTwo.getRecipeSize() < recipeOne.getRecipeSize()) {
          return -1;
       } else {
