--- a/net/minecraft/src/RenderTNTPrimed.java
+++ b/net/minecraft/src/RenderTNTPrimed.java
@@ -9,7 +9,7 @@
       this.field_9246_c = 0.5F;
    }
 
-   public void doRender(EntityTNTPrimed entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
+   public void func_153_a(EntityTNTPrimed entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
       GL11.glPushMatrix();
       GL11.glTranslatef((float)posX, (float)posY, (float)posZ);
       if ((float)entity.fuse - partialTickTime + 1.0F < 10.0F) {
