--- a/net/minecraft/src/RenderArrow.java
+++ b/net/minecraft/src/RenderArrow.java
@@ -3,7 +3,7 @@
 import org.lwjgl.opengl.GL11;
 
 public class RenderArrow extends Render {
-   public void doRender(EntityArrow entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
+   public void renderArrow(EntityArrow entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
       this.loadTexture("/item/arrows.png");
       GL11.glPushMatrix();
       GL11.glTranslatef((float)posX, (float)posY, (float)posZ);
