--- a/net/minecraft/src/RenderSnowball.java
+++ b/net/minecraft/src/RenderSnowball.java
@@ -3,7 +3,7 @@
 import org.lwjgl.opengl.GL11;
 
 public class RenderSnowball extends Render {
-   public void doRender(EntitySnowball entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
+   public void func_155_a(EntitySnowball entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
       GL11.glPushMatrix();
       GL11.glTranslatef((float)posX, (float)posY, (float)posZ);
       GL11.glEnable(32826);
