--- a/net/minecraft/src/RenderMinecart.java
+++ b/net/minecraft/src/RenderMinecart.java
@@ -10,7 +10,7 @@
       this.modelBase = new ModelMinecart();
    }
 
-   public void doRender(EntityMinecart entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
+   public void func_152_a(EntityMinecart entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
       GL11.glPushMatrix();
       double d0 = entity.lastTickPosX + (entity.posX - entity.lastTickPosX) * (double)partialTickTime;
       double d1 = entity.lastTickPosY + (entity.posY - entity.lastTickPosY) * (double)partialTickTime;
