--- a/net/minecraft/src/RenderSpider.java
+++ b/net/minecraft/src/RenderSpider.java
@@ -8,11 +8,11 @@
       this.setRenderPassModel(new ModelSpider());
    }
 
-   protected float getDeathMaxRotation(EntitySpider entityLiving) {
+   protected float setSpiderDeathMaxRotation(EntitySpider entityLiving) {
       return 180.0F;
    }
 
-   protected boolean shouldRenderPass(EntitySpider entityLiving, int pInt) {
+   protected boolean setSpiderEyeBrightness(EntitySpider entityLiving, int pInt) {
       if (pInt != 0) {
          return false;
       } else if (pInt != 0) {
