--- a/net/minecraft/src/RenderManager.java
+++ b/net/minecraft/src/RenderManager.java
@@ -1,6 +1,7 @@
 package net.minecraft.src;
 
 import java.util.HashMap;
+import java.util.Iterator;
 import java.util.Map;
 import org.lwjgl.opengl.GL11;
 
@@ -48,10 +49,10 @@
       this.entityRenderMap.put(EntityBoat.class, new RenderBoat());
       this.entityRenderMap.put(EntityFish.class, new RenderFish());
 
-      for(Render render : this.entityRenderMap.values()) {
-         render.setRenderManager(this);
+      Render render;
+      for(Iterator iterator = this.entityRenderMap.values().iterator(); iterator.hasNext(); render.setRenderManager(this)) {
+         render = (Render)iterator.next();
       }
-
    }
 
    public Render getEntityClassRenderObject(Class entityClass) {
