--- a/net/minecraft/src/RenderLiving.java
+++ b/net/minecraft/src/RenderLiving.java
@@ -15,7 +15,7 @@
       this.renderPassModel = renderPassModel;
    }
 
-   public void doRender(EntityLiving entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
+   public void doRenderLiving(EntityLiving entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
       GL11.glPushMatrix();
       GL11.glDisable(2884);
       this.mainModel.onGround = this.renderSwingProgress(entity, partialTickTime);
