--- a/net/minecraft/src/RenderPainting.java
+++ b/net/minecraft/src/RenderPainting.java
@@ -6,7 +6,7 @@
 public class RenderPainting extends Render {
    private Random rand = new Random();
 
-   public void doRender(EntityPainting entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
+   public void func_158_a(EntityPainting entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
       this.rand.setSeed(187L);
       GL11.glPushMatrix();
       GL11.glTranslatef((float)posX, (float)posY, (float)posZ);
