--- a/net/minecraft/src/RenderSlime.java
+++ b/net/minecraft/src/RenderSlime.java
@@ -10,7 +10,7 @@
       this.scaleAmount = scaleAmount;
    }
 
-   protected boolean shouldRenderPass(EntitySlime entityLiving, int pInt) {
+   protected boolean renderSlimePassModel(EntitySlime entityLiving, int pInt) {
       if (pInt == 0) {
          this.setRenderPassModel(this.scaleAmount);
          GL11.glEnable(2977);
@@ -27,7 +27,7 @@
       }
    }
 
-   protected void preRenderCallback(EntitySlime entityLiving, float partialTickTime) {
+   protected void scaleSlime(EntitySlime entityLiving, float partialTickTime) {
       float f = (entityLiving.field_767_b + (entityLiving.field_768_a - entityLiving.field_767_b) * partialTickTime)
          / ((float)entityLiving.slimeSize * 0.5F + 1.0F);
       float f1 = 1.0F / (f + 1.0F);