--- a/net/minecraft/src/RenderChicken.java
+++ b/net/minecraft/src/RenderChicken.java
@@ -5,11 +5,11 @@
       super(renderBlocks, shadowSize);
    }
 
-   public void doRenderLiving(EntityChicken entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
+   public void renderChicken(EntityChicken entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
       super.doRenderLiving(entity, posX, posY, posZ, yaw, partialTickTime);
    }
 
-   protected float handleRotationFloat(EntityChicken entityLiving, float pFloat) {
+   protected float getWingRotation(EntityChicken entityLiving, float pFloat) {
       float f = entityLiving.field_756_e + (entityLiving.field_752_b - entityLiving.field_756_e) * pFloat;
       float f1 = entityLiving.field_757_d + (entityLiving.destPos - entityLiving.field_757_d) * pFloat;
       return (MathHelper.sin(f) + 1.0F) * f1;
