--- a/net/minecraft/src/RenderPlayer.java
+++ b/net/minecraft/src/RenderPlayer.java
@@ -12,7 +12,7 @@
       super(new ModelBiped(0.0F), 0.5F);
    }
 
-   protected boolean shouldRenderPass(EntityPlayer entityLiving, int pInt) {
+   protected boolean setArmorModel(EntityPlayer entityLiving, int pInt) {
       ItemStack itemstack = entityLiving.inventory.armorItemInSlot(3 - pInt);
       if (itemstack != null) {
          Item item = itemstack.getItem();
@@ -35,7 +35,7 @@
       return false;
    }
 
-   public void doRender(EntityPlayer entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
+   public void renderPlayer(EntityPlayer entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
       ItemStack itemstack = entity.inventory.getCurrentItem();
       this.modelArmorChestplate.heldItemRight = this.modelArmor.heldItemRight = this.modelBipedMain.heldItemRight = itemstack != null;
       this.modelArmorChestplate.isSneak = this.modelArmor.isSneak = this.modelBipedMain.isSneak = entity.isSneaking();
@@ -113,7 +113,7 @@
 
    }
 
-   protected void renderEquippedItems(EntityPlayer entityLiving, float partialTickTime) {
+   protected void renderSpecials(EntityPlayer entityLiving, float partialTickTime) {
       ItemStack itemstack = entityLiving.inventory.armorItemInSlot(3);
       if (itemstack != null && itemstack.getItem().shiftedIndex < 256) {
          GL11.glPushMatrix();
@@ -171,7 +171,7 @@
 
    }
 
-   protected void preRenderCallback(EntityPlayer entityLiving, float partialTickTime) {
+   protected void renderPlayerScale(EntityPlayer entityLiving, float partialTickTime) {
       float f = 0.9375F;
       GL11.glScalef(f, f, f);
    }
