--- a/net/minecraft/src/RenderSorter.java
+++ b/net/minecraft/src/RenderSorter.java
@@ -9,7 +9,7 @@
       this.baseEntity = entityPlayer;
    }
 
-   public int compare(WorldRenderer worldRenderer, WorldRenderer worldRenderer1) {
+   public int doCompare(WorldRenderer worldRenderer, WorldRenderer worldRenderer1) {
       boolean flag = worldRenderer.isInFrustum;
       boolean flag1 = worldRenderer1.isInFrustum;
       if (flag && !flag1) {
