--- a/net/minecraft/src/RenderSpider.java
+++ b/net/minecraft/src/RenderSpider.java
@@ -8,11 +8,11 @@
       this.func_4013_a(new ModelSpider());
    }
 
-   protected float func_172_a(EntitySpider entityLiving) {
+   protected float func_191_a(EntitySpider entityLiving) {
       return 180.0F;
    }
 
-   protected boolean func_166_a(EntitySpider entityLiving, int pInt) {
+   protected boolean func_190_a(EntitySpider entityLiving, int pInt) {
       if (pInt != 0) {
          return false;
       } else if (pInt != 0) {
