--- a/net/minecraft/src/RenderCreeper.java
+++ b/net/minecraft/src/RenderCreeper.java
@@ -7,7 +7,7 @@
       super(new ModelCreeper(), 0.5F);
    }
 
-   protected void preRenderCallback(EntityCreeper entityLiving, float pFloat) {
+   protected void updateCreeperScale(EntityCreeper entityLiving, float pFloat) {
       float f = entityLiving.setCreeperFlashTime(pFloat);
       float f1 = 1.0F + MathHelper.sin(f * 100.0F) * f * 0.01F;
       if (f < 0.0F) {
@@ -25,7 +25,7 @@
       GL11.glScalef(f2, f3, f2);
    }
 
-   protected int getColorMultiplier(EntityCreeper entityLiving, float pFloat, float pFloat1) {
+   protected int updateCreeperColorMultiplier(EntityCreeper entityLiving, float pFloat, float pFloat1) {
       float f = entityLiving.setCreeperFlashTime(pFloat1);
       if ((int)(f * 10.0F) % 2 == 0) {
          return 0;
