--- a/net/minecraft/src/RenderZombieSimple.java
+++ b/net/minecraft/src/RenderZombieSimple.java
@@ -10,7 +10,7 @@
       this.scale = scale;
    }
 
-   protected void preRenderCallback(EntityZombieSimple entityLiving, float partialTickTime) {
+   protected void preRenderScale(EntityZombieSimple entityLiving, float partialTickTime) {
       GL11.glScalef(this.scale, this.scale, this.scale);
    }
 
