--- a/net/minecraft/src/RenderFireball.java
+++ b/net/minecraft/src/RenderFireball.java
@@ -3,7 +3,7 @@
 import org.lwjgl.opengl.GL11;
 
 public class RenderFireball extends Render {
-   public void doRender(EntityFireball entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
+   public void func_4012_a(EntityFireball entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
       GL11.glPushMatrix();
       GL11.glTranslatef((float)posX, (float)posY, (float)posZ);
       GL11.glEnable(32826);
