--- a/net/minecraft/src/RenderPig.java
+++ b/net/minecraft/src/RenderPig.java
@@ -6,7 +6,7 @@
       this.setRenderPassModel(renderPassModel);
    }
 
-   protected boolean shouldRenderPass(EntityPig entityLiving, int pInt) {
+   protected boolean renderSaddledPig(EntityPig entityLiving, int pInt) {
       this.loadTexture("/mob/saddle.png");
       return pInt == 0 && entityLiving.rideable;
    }
