--- a/net/minecraft/src/RenderFallingSand.java
+++ b/net/minecraft/src/RenderFallingSand.java
@@ -9,7 +9,7 @@
       this.shadowSize = 0.5F;
    }
 
-   public void doRender(EntityFallingSand entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
+   public void doRenderFallingSand(EntityFallingSand entity, double posX, double posY, double posZ, float yaw, float partialTickTime) {
       GL11.glPushMatrix();
       GL11.glTranslatef((float)posX, (float)posY, (float)posZ);
       this.loadTexture("/terrain.png");
