--- a/net/minecraft/src/RenderSheep.java
+++ b/net/minecraft/src/RenderSheep.java
@@ -6,7 +6,7 @@
       this.setRenderPassModel(renderPassModel);
    }
 
-   protected boolean shouldRenderPass(EntitySheep entityLiving, int pInt) {
+   protected boolean setWoolColorAndRender(EntitySheep entityLiving, int pInt) {
       this.loadTexture("/mob/sheep_fur.png");
       return pInt == 0 && !entityLiving.sheared;
    }
