--- a/net/minecraft/src/SoundPool.java
+++ b/net/minecraft/src/SoundPool.java
@@ -17,6 +17,7 @@
 
    public SoundPoolEntry addSound(String soundName, File soundFile) {
       try {
+         String s = soundName;
          soundName = soundName.substring(0, soundName.indexOf("."));
          if (this.field_1657_b) {
             while(Character.isDigit(soundName.charAt(soundName.length() - 1))) {
@@ -29,7 +30,7 @@
             this.nameToSoundPoolEntriesMapping.put(soundName, new ArrayList());
          }
 
-         SoundPoolEntry soundpoolentry = new SoundPoolEntry(soundName, soundFile.toURI().toURL());
+         SoundPoolEntry soundpoolentry = new SoundPoolEntry(s, soundFile.toURI().toURL());
          ((List)this.nameToSoundPoolEntriesMapping.get(soundName)).add(soundpoolentry);
          this.allSoundPoolEntries.add(soundpoolentry);
          ++this.numberOfSoundPoolEntries;
