--- a/net/minecraft/src/TexturePackList.java
+++ b/net/minecraft/src/TexturePackList.java
@@ -2,10 +2,8 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+
 import net.minecraft.client.Minecraft;
 
 public class TexturePackList {
@@ -80,8 +78,10 @@
       }
 
       this.availableTexturePacks.removeAll(arraylist);
+      Iterator iterator = this.availableTexturePacks.iterator();
 
-      for(TexturePackBase texturepackbase : this.availableTexturePacks) {
+      while(iterator.hasNext()) {
+         TexturePackBase texturepackbase = (TexturePackBase)iterator.next();
          texturepackbase.unbindThumbnailTexture(this.mc);
          this.field_6538_d.remove(texturepackbase.texturePackID);
       }
