--- a/net/minecraft/src/TexturePackList.java
+++ b/net/minecraft/src/TexturePackList.java
@@ -2,10 +2,8 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+
 import net.minecraft.client.Minecraft;
 
 public class TexturePackList {
@@ -81,9 +79,10 @@
 
       this.availableTexturePacks.removeAll(arraylist);
 
-      for(TexturePackBase texturepackbase : this.availableTexturePacks) {
-         texturepackbase.unbindThumbnailTexture(this.mc);
-         this.field_6538_d.remove(texturepackbase.texturePackID);
+      TexturePackBase texturepackbase;
+      for (Iterator iterator = availableTexturePacks.iterator(); iterator.hasNext(); field_6538_d.remove(texturepackbase.texturePackID)) {
+         texturepackbase = (TexturePackBase) iterator.next();
+         texturepackbase.unbindThumbnailTexture(mc);
       }
 
       this.availableTexturePacks = arraylist;
