--- a/net/minecraft/src/TileEntityRenderer.java
+++ b/net/minecraft/src/TileEntityRenderer.java
@@ -1,6 +1,7 @@
 package net.minecraft.src;
 
 import java.util.HashMap;
+import java.util.Iterator;
 import java.util.Map;
 import org.lwjgl.opengl.GL11;
 
@@ -23,11 +24,11 @@
    private TileEntityRenderer() {
       this.specialRendererMap.put(TileEntitySign.class, new TileEntitySignRenderer());
       this.specialRendererMap.put(TileEntityMobSpawner.class, new TileEntityMobSpawnerRenderer());
+      TileEntitySpecialRenderer tileentityspecialrenderer;
 
-      for(TileEntitySpecialRenderer tileentityspecialrenderer : this.specialRendererMap.values()) {
-         tileentityspecialrenderer.setTileEntityRenderer(this);
+      for(Iterator iterator = this.specialRendererMap.values().iterator(); iterator.hasNext(); tileentityspecialrenderer.setTileEntityRenderer(this)) {
+         tileentityspecialrenderer = (TileEntitySpecialRenderer)iterator.next();
       }
-
    }
 
    public TileEntitySpecialRenderer getSpecialRendererForClass(Class tileEntityClass) {
