--- a/net/minecraft/src/TileEntityRenderer.java
+++ b/net/minecraft/src/TileEntityRenderer.java
@@ -1,6 +1,7 @@
 package net.minecraft.src;
 
 import java.util.HashMap;
+import java.util.Iterator;
 import java.util.Map;
 import org.lwjgl.opengl.GL11;
 
@@ -23,8 +24,10 @@
    private TileEntityRenderer() {
       this.specialRendererMap.put(TileEntitySign.class, new TileEntitySignRenderer());
       this.specialRendererMap.put(TileEntityMobSpawner.class, new TileEntityMobSpawnerRenderer());
+      Iterator iterator = this.specialRendererMap.values().iterator();
 
-      for(TileEntitySpecialRenderer tileentityspecialrenderer : this.specialRendererMap.values()) {
+      while(iterator.hasNext()) {
+         TileEntitySpecialRenderer tileentityspecialrenderer = (TileEntitySpecialRenderer)iterator.next();
          tileentityspecialrenderer.setTileEntityRenderer(this);
       }
 
