--- a/net/minecraft/src/TileEntitySignRenderer.java
+++ b/net/minecraft/src/TileEntitySignRenderer.java
@@ -5,7 +5,7 @@
 public class TileEntitySignRenderer extends TileEntitySpecialRenderer {
    private SignModel modelSign = new SignModel();
 
-   public void renderTileEntityAt(TileEntitySign tileEntity, double coordX, double coordY, double coordZ, float pFloat) {
+   public void renderTileEntitySignAt(TileEntitySign tileEntity, double coordX, double coordY, double coordZ, float pFloat) {
       Block block = tileEntity.getBlockType();
       GL11.glPushMatrix();
       float f = 0.6666667F;
