--- a/net/minecraft/src/TileEntityMobSpawnerRenderer.java
+++ b/net/minecraft/src/TileEntityMobSpawnerRenderer.java
@@ -7,7 +7,7 @@
 public class TileEntityMobSpawnerRenderer extends TileEntitySpecialRenderer {
    private Map entityHashMap = new HashMap();
 
-   public void renderTileEntityAt(TileEntityMobSpawner tileEntity, double posX, double posY, double posZ, float pFloat) {
+   public void renderTileEntityMobSpawner(TileEntityMobSpawner tileEntity, double posX, double posY, double posZ, float pFloat) {
       GL11.glPushMatrix();
       GL11.glTranslatef((float)posX + 0.5F, (float)posY, (float)posZ + 0.5F);
       Entity entity = (Entity)this.entityHashMap.get(tileEntity.mobID);
