--- a/net/minecraft/src/World.java
+++ b/net/minecraft/src/World.java
@@ -6,12 +6,7 @@
 import java.io.FileInputStream;
 import java.io.FileOutputStream;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Random;
-import java.util.Set;
-import java.util.TreeSet;
+import java.util.*;
 
 public class World implements IBlockAccess {
    public boolean scheduledUpdatesAreImmediate = false;
@@ -1580,7 +1575,8 @@
          --this.ambientTickCountdown;
       }
 
-      for(ChunkCoordIntPair chunkcoordintpair : this.activeChunkSet) {
+      for(Iterator iterator = this.activeChunkSet.iterator(); iterator.hasNext(); ) {
+         ChunkCoordIntPair chunkcoordintpair = (ChunkCoordIntPair)iterator.next();
          int i2 = chunkcoordintpair.chunkXPos * 16;
          int j2 = chunkcoordintpair.chunkZPos * 16;
          Chunk chunk = this.getChunkFromChunkCoords(chunkcoordintpair.chunkXPos, chunkcoordintpair.chunkZPos);
